### ISSUE TITLE
v0.4.7 does not output errors

### ISSUE THREAD

**Issue by gahrae** [apigee/apigeelint#175]
Version 0.4.7 does not report errors.

**Steps to replicate:**

0. Introduce an error, for example missing semi-colon in JS file.
1. Set version to 0.4.7 in package.json
2. npm install
3. Verify version:
$ node ./node_modules/apigeelint/cli.js -V
0.4.7

4. Call cli script directly:
$ node ./node_modules/apigeelint/cli.js -s ./apiproxy -f table

**OUTPUT**
0 Errors
0 Warnings

---

Also affects global installation:
$ apgeelint -V
0.4.7

$ apigeelint -s ./apiproxy -f table

**OUTPUT**
0 Errors
0 Warnings

---

**Expected output (when using version 0.4.6)**

Remove current version
$ rm -rf ./node_modules/apigeelint

Change to version 0.4.6
$ vi package.json
$ grep apigeelint package.json
    "apigeelint": "0.4.6",
$ npm install

Run again
$ node ./node_modules/apigeelint/cli.js -s ./apiproxy -f table

**OUTPUT**
redacted_file_name

Line, Column, Type, Message,Rule ID
7,59,error,semi: ExpressionStatement: Missing semicolon.,PO025
8,59,error,(error): Missing semicolon.,PO013
36,1,error,(error): Misleading line break before '||'; readers may interpret this as an expression  boundary,PO013

3 Errors
0 Warnings

---

Note: Only text was shown from the output. The actual 'table' output includes extra characters for formatting.

**Comment by gahrae** [comment]
Bug introduced with: https://github.com/apigee/apigeelint/commit/ee2871eb7e0444bd018fdda85d974acaef1d6f52

**Comment by ssvaidyanathan** [comment]
let me try and revert that :(

**Comment by ssvaidyanathan** [comment]
Just published 0.4.8
Can you please try

**Comment by gahrae** [comment]
**Fix verified.**

**Verification steps:**

```
rm -rf node_modules/apigeelint/
vi package.json
grep apigeelint package.json
    "apigeelint": "0.4.8",
npm install
node ./node_modules/apigeelint/cli.js -V
    0.4.8
node ./node_modules/apigeelint/cli.js -s ./apiproxy -f table
```

**Output as expected:**
```
Line, Column, Type, Message, Rule ID
20, 45, error, semi: ExpressionStatement: Missing semicolon., PO025
21, 45, error, (error): Missing semicolon., PO013

2 Errors
0 Warnings
```

Thanks!

**Comment by ssvaidyanathan** [comment]
Thanks for checking and helping here.. My bad..
